tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -1579,7 +1579,7 @@ def get_placeholder_shapes(argv_input: str, argv_input_shape: str, argv_batch=No
         # clean inputs from values for freezing
         inputs_without_value = list(map(lambda x: x.split('->')[0], inputs))
         placeholder_shapes = dict(zip_longest(inputs_without_value,
-                                              map(lambda x: PartialShape(x) if x else None, shapes)))
+                                              map(lambda x: PartialShape(x) if (x or x == '') else None, shapes)))
```

Comment:
why is this change needed? Please test this case as well.

Reply:
This line is needed exactly for the cases with inputs with scalars and all new test cases test this line as well. E.g. if there are inputs `[12,4,1],[]` they will be split into 2 values with `'12,4,1'` and `''` and for the later case we need not `None` but `PartialShape([])` or equivalent `PartialShape('')`

![image](https://github.com/openvinotoolkit/openvino/assets/5703039/8c4e5502-c926-4747-b42a-0f06e6b1bc00)